### PR TITLE
Read only flag

### PR DESCRIPTION
**Pull request by someotherself**
New --read-only flag added.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -43,8 +42,7 @@
 //!     args.next(); // skip program name
 //!     let mount_path = args.next().expect("mount_path expected");
 //!     let data_path = args.next().expect("data_path expected");
-//!
-//!     struct PasswordProviderImpl {}
+//!     use tracing::info;struct PasswordProviderImpl {}
```
Comment (radumarias) [inline src/lib.rs]: looks kinda strange in one line

File: src/run.rs [hunk 2]
```diff
@@ -177,6 +177,14 @@ fn get_cli_args() -> ArgMatches {
                         .action(ArgAction::SetTrue)
                         .help("If it should allow setting SUID and SGID when files are created. Default is false and it will unset those flags when creating files"),
                 )
+                .arg(
+                    Arg::new("read-only")
+                        .long("read-only")
+                        .short('e')
+                        .action(ArgAction::SetTrue)
+                        .requires("mount-point")
```
Comment (radumarias) [inline src/run.rs:196]: also `.requires("data-dir")`

**Review by radumarias — CHANGES_REQUESTED** [review]

File: src/lib.rs (hunk 1 above)
Comment (someotherself) [reply src/lib.rs]: Oups. Fixed.

File: src/run.rs (hunk 2 above)
Comment (someotherself) [reply src/run.rs:196]: It's now added.

File: src/lib.rs [hunk 3]
```diff
@@ -43,7 +42,7 @@
 //!     args.next(); // skip program name
 //!     let mount_path = args.next().expect("mount_path expected");
 //!     let data_path = args.next().expect("data_path expected");
-//!
+//!     use tracing::info;
```
Comment (radumarias) [inline src/lib.rs]: why don't you leave the here? https://github.com/radumarias/rencfs/pull/167/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759L27

**Review by radumarias — CHANGES_REQUESTED** [review]

File: src/lib.rs (hunk 3 above)
Comment (someotherself) [reply src/lib.rs]: Fixed.

**Review by radumarias — APPROVED** [review]

**Comment by radumarias** [comment]
merged

**Pull request closed without merging**